packages/common/string.js

```diff
@@ -52,6 +52,22 @@ function globalReplace(string, oldSubString, newSubString) {
   return string.replace(new RegExp(oldSubString, 'g'), newSubString);
 }
 
+/**
+ * Globally replaces substr in string with newSubstr
+ *
+ * @param {string} substr - The string to replace
+ * @param {string} newSubstr - The string replacement
+ * @param {string} string - The string to modify
+ * @returns {string} the modified string
+ *
+ * This is a curried function - https://lodash.com/docs/4.17.11#curry
+ *
+ * @static
+ */
+const globalReplaceC = curry(
+  (substr, newSubstr, string) => globalReplace(string, substr, newSubstr)
```

Comment:
As annoying as more typing is, I'm not immediately on-board with the "C" as "curried" function convention.     Consider `globalReplaceCurried` for readability instead. 

Also consider deprecating `globalReplace`, or just wholesale updating it.   We can always publish it as a breaking change in the CL, *particularly* given TF is going to be a major release.